.eslintrc.yml

```diff
@@ -53,3 +53,6 @@ rules:
   jsx-a11y/anchor-is-valid:
     - error
     - specialLink: ["to"]
+
+  react/no-array-index-key:
```

Comment:
it's not recommended to use array index as key. If some special occasions we really need to do this, just override the rule temporary, not globally.